src/Server/HTTPHandler.cpp

```diff
@@ -595,6 +595,8 @@ void HTTPHandler::processQuery(
 
     customizeContext(request, context);
 
+    query_scope.emplace(context);
```

Comment:
Does it mean, it will deallocate untrack more than track?
Ok.

Reply:
It will be tracked the memory tracker correctly, since ThreadStatus is already created.
Later QueryScope should not affect anything significant